cmd/gossamer/commands/account.go

```diff
@@ -24,15 +24,13 @@ var AccountCmd = &cobra.Command{
 	Use:   "account",
 	Short: "Create and manage node keystore accounts",
 	Long: `The account command is used to manage the gossamer keystore.
-Examples: 
+Examples:
 
-To generate a new sr25519 account: 
-	gossamer account generate --keystore-path=path/to/location --scheme=sr25519
-To generate a new ed25519 account: 
-	gossamer account generate --ed25519
-To generate a new secp256k1 account: 
+To generate a new ed25519 account:
+	gossamer account generate --keystore-path=path/to/location --scheme=ed25519
```

Comment:
I took the freedom to change the existing example to use ed25519 instead of sr25519 since the later it's the default one, but if it don't make sense I just rollback and remove the command.

Reply:
I think that `cobra` already does that, probably because `crypto.Sr25519Type` it's passed instead of a empty string, for example, in the other commands it's not and it does not output `defaults ...`.

![image](https://github.com/ChainSafe/gossamer/assets/44412643/2295f26b-7260-4e07-bd8e-30bef78b2f99)

Reply:
I confirmed, the second argument of `Flags().String()` it's the default value.

Reply:
Thank you for being thorough!